src/Emanote/Model/Stork/Index.hs

```diff
@@ -98,7 +113,36 @@ fileCodec =
     <*> Toml.text "url" .= fileUrl
     <*> Toml.text "title" .= fileTitle
 
+showHandling :: Handling -> Text
+showHandling handling = case handling of
+  Handling_Ignore -> "Ignore"
+  Handling_Omit -> "Omit"
+  Handling_Parse -> "Parse"
+
+parseHandling :: Text -> Either Text Handling
+parseHandling handling = case handling of
+  "Ignore" -> Right Handling_Ignore
+  "Omit" -> Right Handling_Omit
+  "Parse" -> Right Handling_Parse
+  other -> Left $ "Unsupported value for frontmatter handling: " <> other
+
+handlingCodec :: Toml.Key -> TomlCodec Handling
+handlingCodec = textBy showHandling parseHandling
+
 inputCodec :: TomlCodec Input
 inputCodec =
   Input
-    <$> Toml.list fileCodec "input.files" .= inputFiles
+    <$> Toml.list fileCodec "files" .= inputFiles
+    <*> Toml.diwrap (handlingCodec "frontmatter_handling") .= inputFrontmatterHandling
+
+storkInputCodec :: TomlCodec StorkInput
+storkInputCodec =
+  StorkInput
+    <$> Toml.table inputCodec "input" .= globalInput
+
+instance FromJSON Handling where
+  parseJSON = Aeson.withText "FrontmatterHandling" $ \case
+    "ignore" -> pure Handling_Ignore
+    "omit" -> pure Handling_Omit
+    "parse" -> pure Handling_Parse
+    _ -> fail "Unsupported value for frontmatter-handling"
```

Comment:
Is there no way to implement this instance in terms of `Show` instance?

Reply:
This would be in terms of the `Read` instance, no?

Reply:
Stork needs the string capitalized. To reuse the instance, we would need to take the value capitalized in the YAML as well.

Reply:
I think there should be a way to do this using generics. Aeson has generic deriving, even with options.

Reply:
I see. I haven't read about generics yet. I'll see how it goes.

Reply:
I was able to derive the `FromJSON` instance generically, but Tomland doesn't support generic derive for sum types. I left the current implementation for now.